### ISSUE TITLE
PNI: update homepage copy design

### ISSUE THREAD

**Issue by kristinashu** [mozilla/foundation.mozilla.org#3751]
Need copy from @caltrider then we will mock it up and give to devs https://github.com/mozilla/foundation.mozilla.org/issues/3752.

**Comment by beccaklam** [comment]
@caltrider Can I get some copy for this?

**Comment by beccaklam** [comment]
Jen to keep thinking on this, but feels that the "holiday" will mostly come from the banner.

**Comment by kristinashu** [comment]
Keeping the copy as is for MozFest but I'll move this ticket to our next milestone in case we want to change it then.

**Comment by kristinashu** [comment]
I think Matthew was hoping we could reduce the amount of copy on the homepage. Maybe we could remove "Look for the “Meets Our Minimum Security Standards” badge to get started."?

**Comment by beccaklam** [comment]
@kristinashu Yeah that could work, though I think he was really looking to get rid of the line after that describing how the face worked.

**Comment by kristinashu** [comment]
@caltrider says she'll have copy for this on Monday.

**Comment by caltrider** [comment]
Here's the updated landing page copy. I tightened it a bit. I thought about losing the line that explains the scrolling creepy face, but decided to keep it based on how people seemed to benefit from an explanation of that at MozFest. 

Here's the updated copy:
Be Smart. Shop Safe.
How creepy is that smart speaker, that fitness tracker, those wireless headphones? We created this guide to help you shop for safe, secure connected products.

The 😮 below shows how creepy users find these products. 
@kristinashu @beccaklam
